content/docs/capabilities/high-availability.mdx

```diff
@@ -122,20 +134,25 @@ Multiple replicas of Databroker or all-in-one service are only supported with [e
 
 :::
 
-## Example
+## Example: Split-Service Mode
 
-The following setup would demonstrate a minimum configuration for the split-service mode, using docker compose on a local host.
+The following configuration demonstrates a minimum configuration for the split-service mode, using Docker Compose on a local host.
 
-This guide intentionally omits provisioning certificates for internal Pomerium interaction for simplicity.
+This guide intentionally omits provisioning certificates for internal Pomerium interactions for simplicity.
 
 <InstallMkcert />
 
-Generate a wildcard certificate and key for Pomerium to use. You should get two files `_wildcard.localhost.pomerium.io-key.pem` and `_wildcard.localhost.pomerium.io.pem`. This is a special domain that always resolves to `127.0.0.1`.
+Generate a wildcard certificate and key for Pomerium to use:
 
 ```sh
 mkcert "*.localhost.pomerium.io"
 ```
 
+This should output two files:
+
+- `_wildcard.localhost.pomerium.io-key.pem`
+- `_wildcard.localhost.pomerium.io.pem` (This is a special domain that always resolves to `127.0.0.1`.)
```

Comment:
(optional)
```suggestion
- `_wildcard.localhost.pomerium.io.pem`

(Our DNS is configured so that all sub-domains of "localhost.pomerium.io" will resolve to the loopback address `127.0.0.1`.)
```